sdk/libmcm_dp.pc.in

```diff
@@ -0,0 +1,14 @@
+prefix="@CMAKE_INSTALL_PREFIX@"
```

Comment:
Out of scope.

Reply:
This should stay with this PR since the FFmpeg MCM plugin directly depends on this .pc file.